src/Console/ConfigurationResolver.php

```diff
@@ -258,15 +260,20 @@ public function getDiffer()
             $mapper = [
                 'null' => function () { return new NullDiffer(); },
                 'sbd' => function () { return new SebastianBergmannDiffer(); },
+                'unidiff' => function () { return new UnifiedDiffer(); },
```

Comment:
is `unidiff` a common name? so far, we used `udiff` in discussion

Reply:
both are common use, but I agree, lets use `udiff`, will update :+1: